src/main/kotlin/me/partlysanestudios/partlysaneskies/utils/vectors/Point3d.kt

```diff
@@ -4,9 +4,16 @@
 //
 package me.partlysanestudios.partlysaneskies.utils.vectors
 
+import me.partlysanestudios.partlysaneskies.PartlySaneSkies
 import net.minecraft.util.BlockPos
 
 open class Point3d(x: Double, y: Double, val z: Double) : Point2d(x, y) {
+    companion object {
+        fun asPlayer(): Point3d {
+            return Point3d(PartlySaneSkies.minecraft.thePlayer?.posX ?: -1.0, PartlySaneSkies.minecraft.thePlayer?.posY ?: -1.0, PartlySaneSkies.minecraft.thePlayer?.posZ ?: -1.0)
```

Comment:
```suggestion
            val player = PartlySaneSkies.minecraft.thePlayer
           return Point3d(player?.posX ?: -1.0, player?.posY ?: -1.0, player?.posZ ?: -1.0)
```

its a bit better like that